### DISCUSSION TITLE
Renewing local EJBCA server SSL

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#380] [General]
Hello everyone, I deployed ejbca8.0 CE through docker with wildfly 26 and using locally generated SSL (under management CA). Now the issue is that it shall expire very soon so how can I renew it.

I can reissue it through adminweb in p12 or JKS format but how can I deploy it on wildfly. Is there any guideline for it

**Comment by primetomas** [comment]
I added some more documentation under the section of using an external database here:
https://hub.docker.com/repository/docker/keyfactor/ejbca-ce/general

For a simple setup like that, simply starting a new container will generate a new TLS server certificate for this instance.
